src/main/java/nextstep/ladder/domain/JoinMembers.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class JoinMembers {
+    private final List<Member> members = new ArrayList<>();
+
+    public JoinMembers(List<Member> members) {
+        this.members.addAll(members);
```

Comment:
```suggestion
        this.members = members;
```

위와 같이 구현하는 것이 낫지 않을까?

Reply:
넵 변경 완료했습니다!